docs/source/DeveloperDocumentation/MobileSdks/ios-fairplay-sdk.md

```diff
@@ -20,7 +20,7 @@ Current release:	iOS v1.0 (254)
 
 ### Overview
 
-VUDRMFairPlay SDK enables Apple’s AVPlayer from AVFoundation to securely request licenses from Vualto’s VUDRM cloud based DRM platform using an extended instance of AssetResourceLoaderDelegate.
+VUDRMFairPlay SDK enables Apple’s AVPlayer from AVFoundation to securely request licenses from JW Player's Studio DRM cloud based DRM platform using an extended instance of AssetResourceLoaderDelegate.
```

Comment:
Are we not able to rebrand the VUDRMFairPlay SDK?

Reply:
Not currently as this is referring to the old SDK's, once @macmagician has finished that rebrand I think we'd be able to update all the SDK docs to be more Studio DRM

Reply:
OK cool!